microprofile/WFLY-12809_EAP7-1373_quickstart-for-mp-metrics-2.2.adoc

```diff
@@ -0,0 +1,66 @@
+= WFLY-12809 / EAP7-1373 Quickstart for MP Metrics 2.2
+:author:            Martin Stefanko
+:email:             mstefank@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+This RFE provides a new quickstart and guide for MicroProfile Metrics 2.2 use in WildFly /
+JBoss EAP. The example will cover all basic functionalities of MicroProfile Metrics 2.2,
+namely: usage of individual application metrics (annotations) and accessing the metrics
+through the management web interface.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/EAP7-1373[EAP7-1373]
+* https://issues.jboss.org/browse/WFLY-12809[WFLY-12809]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1368
+
+=== Dev Contacts
+
+* mailto:mstefank@redhat.com[Martin Stefanko]
+
+=== QE Contacts
+
+* TBD
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+[x] Engineering
+
+[ ] QE
+
+=== Affected Projects or Components
+
+* Quickstarts
+
+=== Other Interested Projects
+
+== Requirements
+
+=== Hard Requirements
+
+* usage of application metrics: @Counted, @Gauge, @Timed, @Metric, @Metered
+* different scopes of metrics: base, vendor, application
```

Comment:
please make sure that the quickstart also mentions the WildFly subsystem metrics that are exposed in the `vendor` scope.